src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -88,11 +88,7 @@ class FormConfigBuilder implements FormConfigBuilderInterface
      */
     private $formFactory;
 
-    /**
-     * @var string|null
-     */
-    private $action;
-
+    private $action = '';
```

Comment:
should be safe and mean the same thing. it's only used in the template where both null and '' are treated the same.